tests/reconfiguration.py

```diff
@@ -341,6 +344,29 @@ def test_retiring_nodes_emit_at_most_one_signature(network, args):
     return network
 
 
+@reqs.description("Adding a learner without snapshot")
+def test_learner_catches_up(network, args):
```

Comment:
We should check that the new node doesn't count in the election quorum or for commit. I think the best way for this would be to suspend a majority of followers (expect the new node) and make sure the commit is stuck, and that no new node can be elected as primary when the primary is suspended/killed.

Reply:
Both added!